packages/devextreme/js/__internal/ui/splitter/splitter.ts

```diff
@@ -308,10 +319,10 @@ class Splitter extends CollectionWidget<Properties> {
       const $rightItem = this._getResizeHandleRightItem($resizeHandle);
       const leftItemData = this._getItemData($leftItem);
       const rightItemData = this._getItemData($rightItem);
-      const resizable = leftItemData.resizable !== false
-        && rightItemData.resizable !== false
-        && leftItemData.collapsed !== true
-        && rightItemData.collapsed !== true;
+      const resizable = leftItemData?.resizable !== false
```

Comment:
Why is it necessary to check that left or right items are present? I believe that resizeHandle shouldn't exist if left or right items are missing.